src/services/ContextualIdentitiesEvents.ts

```diff
@@ -0,0 +1,161 @@
+/**
+ * Copyright (c) 2020 Kenneth Tran and CAD Team (https://github.com/Cookie-AutoDelete/Cookie-AutoDelete/graphs/contributors)
+ * Licensed under MIT (https://github.com/Cookie-AutoDelete/Cookie-AutoDelete/blob/3.X.X-Branch/LICENSE)
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+import StoreUser from './StoreUser';
+import { cacheCookieStoreIdNames, removeListUI } from '../redux/Actions';
+import contextualIdentitiesChangeInfo = browser.contextualIdentities.contextualIdentitiesChangeInfo;
+import { cadLog, eventListenerActions, getSetting } from './Libs';
+import { ReduxConstants } from '../typings/ReduxConstants';
+
+export default class ContextualIdentitiesEvents extends StoreUser {
+  public static init(): void {
+    if (!browser.contextualIdentities) return;
```

Comment:
Feel free to combine the shorter if statements like with this and `ContextualIdentitiesEvents.isInitialized`

Reply:
Combined via 4153672